Client/Frontend/Browser/TabToolbar.swift

```diff
@@ -195,6 +195,7 @@ open class TabToolbarHelper: NSObject {
     }
 
     func didLongPressStopReload(_ recognizer: UILongPressGestureRecognizer) {
+        guard !isSearch else { return }
```

Comment:
Added a guard to not trigger long press for when 🔍 icon is being shown.

Reply:
Added a guard to not trigger long press for when 🔍 icon is being shown.

Reply:
Added a guard to not trigger long press for when 🔍 icon is being shown.